src/main.rs

```diff
@@ -273,32 +281,27 @@ impl<'a> Frame<'a> {
 
     }
 
+    pub fn register_class(&mut self, name: String, methods: HashMap<String,Node>) {
```

Comment:
Why would I use a `Frame` to register a class? Shouldn't that happen on the VM object?

Reply:
Yes that is correct, this was a bit of a stupid way of doing it. However, this has been implicitly fixed in an upcoming PR where this is no longer necessary as the the VM has been changed to a bytecode interpreter.